main.go

```diff
@@ -75,6 +77,9 @@ func setupCollectors(logger log.Logger, configFile string) ([]prometheus.Collect
 			vpcSessions = append(vpcSessions, sess)
 		}
 		vpcExporter := pkg.NewVPCExporter(vpcSessions, logger, config.VpcConfig, awsAccountId)
+
+		pkg.TakeRole(sess, roleARN, sessionName, logger)
```

Comment:
I think takerole should go up as well, since the session is based on the environment variables